pouta_blueprints/static/partials/welcome.html

```diff
@@ -1,10 +1,43 @@
-<div class="jumbotron">
-    <h1><configurable-value key="INSTALLATION_NAME"></configurable-value></h1>
-    <p configurable-show key="ENABLE_SHIBBOLETH_LOGIN">
-        <b>You can also login with</b>
-        <a href="/login"><img src="/img/haka.gif" alt="Haka login" /></a>
-    </p>
-    <p>If you don't have access to your password, you
-    can <a href="#/reset_password">reset it here</a>.
-    </p>
+<div class="jumbotron" ng-controller="NavbarController">
+        <div class="col-md-6 col-xs-6">
+            <h1 name="installation-name"><configurable-value key="INSTALLATION_NAME"></configurable-value></h1>
+            <p name="installation-description"><configurable-value
+                                    key="INSTALLATION_DESCRIPTION"></configurable-value></p>
+        </div>
+    <div class="row">
+        <div class="col-md-6 col-xs-6">
+            <p configurable-show key="ENABLE_SHIBBOLETH_LOGIN">
+                <b>Login with with</b>
```

Comment:
I think we could also just one big logo of HAKA and get rid of the preceding text. Not sure, just a suggestion